CHANGELOG.md

```diff
@@ -24,3 +24,70 @@ bump for bug fixes,
 # 1.0.0.4
 bumped version for float/double range bugfix
 
+# 1.2
+
+1. Breaking change which mostly maintains backwards compatibility.
+2. Faster by more x10 (depending on the type) - see below for benchmarks.
+3. Passes a large number of random number test suites:
+  * [dieharder](http://webhome.phy.duke.edu/~rgb/General/dieharder.php "venerable")
+  * [TestU01 (SmallCrush, Crush, BigCrush)](http://simul.iro.umontreal.ca/testu01/tu01.html "venerable")
+  * [PractRand](http://pracrand.sourceforge.net/ "active")
+  * [gjrand](http://gjrand.sourceforge.net/ "active")
+  * [rademacher-fpl](https://gitlab.com/christoph-conrads/rademacher-fpl/-/tree/master "active")
+  * [gjrand](http://gjrand.sourceforge.net/ "active")
+  * See [random-quality](https://github.com/tweag/random-quality)
+		 for details on how to do this yourself.
+4. Better quality split as judged by these
+	[tests](https://www.cambridge.org/core/journals/journal-of-functional-programming/article/evaluation-of-splittable-pseudorandom-generators/3EBAA9F14939C5BB5560E32D1A132637). Again
+	see [random-quality](https://github.com/tweag/random-quality) for
+	details on how to do this yourself.
+5. Unbiased generation of ranges.
+6. Updated tests and benchmarks.
+7. [Conntinuous integration](https://travis-ci.org/github/idontgetoutmuch/random).
+8. Fully documented - for more details see the [haddock](https://htmlpreview.github.io/?https://github.com/idontgetoutmuch/random/blob/release-notes/docs/System-Random.html).
```

Comment:
Note that the docs are still WIP - not just because of aesthetics, but they're wrong right now: e.g. `PrimGen` is still documented, but `AtomicGen` isn't. See https://github.com/idontgetoutmuch/random/pull/83.

Reply:
I won't announce anything until these are correct